go-update/src/main/java/org/reactome/release/goupdate/GoTermInstanceModifier.java

```diff
@@ -359,13 +363,76 @@ else if (GoTermsUpdater.getReferrerCountsExcludingGOEntities(this.goInstance).is
 			e.printStackTrace();
 		}
 	}
+
+	/*
+	 * Clears reference attributes that point TO *this* goInstance FROM other GO entities. To be used when an instance is being deleted.
+	 */
+	private void clearAttributesFromReferringGOEntities() throws Exception
+	{
+		Map<GKSchemaAttribute, Integer> goReferrerCounts = GoTermsUpdater.getReferrerCountsFilteredByClass(this.goInstance, GoTermsUpdater.isNotGOEntity.negate());
+		// If there are any referrers that are GO entities...
+		if (goReferrerCounts.size() > 0)
```

Comment:
I don't think this check is necessary.  If the size of `goReferrerCounts` is zero, the call to the keySet method will return an empty set on line 376 and the for loop won't be entered.